AmpFinKit/Sources/AFNetwork/HTTP/JellyfinClient.swift

```diff
@@ -76,6 +76,14 @@ public extension JellyfinClient {
             _userId ?? ""
         }
     }
+
+    var supportsAudioRemuxing: Bool {
+        get {
+            let (major, minor, _) = serverVersion ?? (0,0,0);
+
+            return major >= 10 && minor >= 10;
```

Comment:
Forgive the drive-by PR review, but won't this start returning `false` when version 11.0 comes out? At least until 11.10 comes around.

Reply:
Yes it will. But from Jellyfin server perspective 11.0 means almost each and every api is broken and the clients must have to accommodate that, which means it should be okay for AmpFin.

Reply:
There is already a system to perform these checks: https://github.com/rasmuslos/AmpFin/blob/main/AmpFinKit/Sources/AFNetwork/HTTP/JellyfinClient%2BFeatureFlags.swift